src/components/Todo.jsx

```diff
@@ -0,0 +1,80 @@
+import { useState } from "react";
+import plus from "../assets/plus.svg";
+import Task from "./Task";
+import { nanoid } from "nanoid";
+
+export const Todo = () => {
```

Comment:
You can use default export as it is the only element exported from the file.

Reply:
sure, thanks